keyboards/ortho5by12/rules.mk

```diff
@@ -0,0 +1,45 @@
+# MCU name
+MCU = atmega328p
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
+#   ATmega328P   USBasp
+# This uses usbaspbootloader
+BOOTLOADER = USBasp
+
+
+# Flash program via avrdude, but default command is not suitable.
+# You can use plaid:default:program
+PROGRAM_CMD = avrdude -c usbasp -p m328 -U flash:w:$(BUILD_DIR)/$(TARGET).hex
+
+
+# disable debug code
+OPT_DEFS = -DDEBUG_LEVEL=0
+
+
```

Comment:
```suggestion
```

Reply:
actually there's a mistake there, that should be m328p can you edit it and suggest it to me? I'm still new to github and don't know how to, without starting from scratch again.

Reply:
Doesn't matter, this can be deleted anyway - the `:flash` target should work fine.